docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,639 @@
+          $ref: "#/components/schemas/Id"
+      required:
+        - id
+
+    CreateSignatureRequestBody:
+      type: object
+      properties:
+        dossier_id:
+          $ref: "#/components/schemas/Id"
+        signer_id:
+          allOf:
+            - $ref: "#/components/schemas/Id"
+            - example: "01BX5ZZKBKACTAV9WEVGEMMVS0"
+        expires_at:
+          $ref: "#/components/schemas/Timestamp"
+        notify_user_via_io_message:
+          type: boolean
+          default: false
+      required:
+        - dossier_id
+        - signer_id
+
+    Document:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        metadata:
+          $ref: "#/components/schemas/DocumentMetadata"
+        created_at:
+          $ref: "#/components/schemas/Timestamp"
+        updated_at:
+          $ref: "#/components/schemas/Timestamp"
+      required:
+        - id
+        - status
+        - metadata
+        - created_at
+        - updated_at
+
+    DocumentToBeUploaded:
+      allOf:
+        - $ref: "#/components/schemas/Document"
+        - type: object
+          properties:
+            status:
+              type: string
+              enum:
+                - WAIT_FOR_UPLOAD
+          required:
+            - status
+
+    DocumentToBeValidated:
+      allOf:
+        - $ref: "#/components/schemas/Document"
+        - type: object
+          properties:
+            status:
+              type: string
+              enum:
+                - WAIT_FOR_VALIDATION
+            uploaded_at:
+              $ref: "#/components/schemas/Timestamp"
+          required:
+            - status
+            - uploaded_at
+
+    DocumentUploaded:
+      allOf:
+        - $ref: "#/components/schemas/Document"
+        - type: object
+          properties:
+            status:
+              type: string
+              enum:
+                - READY
+            uploaded_at:
+              $ref: "#/components/schemas/Timestamp"
+            url:
+              type: string
+              format: uri
+          required:
+            - status
+            - uploaded_at
+            - url
+
+    DocumentRejected:
+      allOf:
+        - $ref: "#/components/schemas/Document"
+        - type: object
+          properties:
+            status:
+              type: string
+              enum:
+                - REJECTED
+            uploaded_at:
+              $ref: "#/components/schemas/Timestamp"
+            rejected_at:
+              $ref: "#/components/schemas/Timestamp"
+            reject_reason:
+              type: string
+          required:
+            - status
+            - uploaded_at
+            - rejected_at
+            - reject_reason
+
+    SignatureRequest:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        dossier_id:
+          $ref: "#/components/schemas/Id"
+        expires_at:
+          $ref: "#/components/schemas/Timestamp"
```

Comment:
`signer_id` is missing

Reply:
After giving it some thought, I decided it wasn't necessary to include `signer_id` in this response, maybe `dossier_id` is redundant either. Do you believe it should be included?

Reply:
I think it is useful as the `ISSUER` given a `signature_request_id` will no longer be able to trace the `signer_id` if he has not stored this information somewhere.